Modules/HRV_DFA/DFA.cpp

```diff
@@ -0,0 +1,145 @@
+#include "DFA.h"
+
+using namespace std;
+
+void polyfit::calculateCoefficient(vector<int>& x)
+    {
+        double N = x.size();
+        double numerator
+            = (N * sum_xy - sum_x * sum_y);
+        double denominator
+            = (N * sum_x_square - sum_x * sum_x);
+        coeff = numerator / denominator;
+    }
+
+void polyfit::calculateConstantTerm(vector<int>& x)
+    {
+        double N = x.size();
+        double numerator
+            = (sum_y * sum_x_square - sum_x * sum_xy);
+        double denominator
+            = (N * sum_x_square - sum_x * sum_x);
+        constTerm = numerator / denominator;
+    }
+
+int polyfit::sizeOfData(vector<int>& x)
+    {
+        return x.size();
+    }
+
+double polyfit::coefficient(vector<int>& x)
+    {
+        if (coeff == 0)
```

Comment:
Dlaczego takie porównanie? Może on po prostu ma wynosić 0 dla niektórych przypadków

Reply:
to już usunięte